.dagger/README.md

```diff
@@ -12,117 +17,144 @@ A module is a collection of functions (and types) which can be used by `dagger c
 
 To discover available functions in a given module: `dagger functions`.
 
-With that in mind: this document includes examples of *typical pipelines* that are useful while developing Dagger. But remember, you are free to compose your own pipelines, either by modifying the examples, of starting from scratch. This flexibility is one of the killer features of Dagger.
+```sh {"id":"01J5Y82842KAER30RGCN96HH6M"}
```

Comment:
:eyes: what are these IDs?

Does editing this doc then mean that adding new codeblocks *requires* the use of the vscode extension to edit it? If so, that feels quite overly restrictive, and limits who can edit this.

---

Aside from that, not really any strong opinions on the change, I'm not sure what value this provides (it's really just a cheatsheet for devs), but also it's not overly invasive, so :woman_shrugging:

Reply:
> 👀 what are these IDs?
> 
> Does editing this doc then mean that adding new codeblocks _requires_ the use of the vscode extension to edit it? If so, that feels quite overly restrictive, and limits who can edit this.

Nobody's required to use Runme CLI/extension for editing or viewing - it's just markdown. The IDs are optional, and I should have turned them off. They easily evaded my selective perception because I'm so used to seeing them.

> could potentially be interesting for `RELEASING.md` in it's current state

I'm happy to help if you ever decide to take a stab at it.